src/components/Centralizer/Centralizer.css

```diff
@@ -0,0 +1,4 @@
+.Centralizer {
+  width: 1440px;
+  margin: auto 0;
```

Comment:
@ViniciusDaCunha hey please uso margin: 0 auto to centralize horizontally

Reply:
ok